test/verify/check-machines-dbus

```diff
@@ -301,7 +347,7 @@ class TestMachinesDBus(machineslib.TestMachines):
 
         # Check overview network properties are present
         b.wait_in_text("#network-test_network3-{0}-persistent".format(connectionName), "no")
-        b.wait_in_text("#network-test_network3-{0}-autostart".format(connectionName), "no")
+        b.wait_not_present("#network-test_network2-{0}-autostart-checkbox:checked".format(connectionName))
```

Comment:
Here you replaced network-3 with network-2. And my previous comment proves it self since this check became kinda useless.